app/assets/stylesheets/_category.scss

```diff
@@ -5,6 +7,17 @@
   margin-left: 15px;
   margin-right: 15px;
 
+  li{
+    @include border-radius(5px);
+    border:1px solid white;
+    color:white;
+    padding:10px;
+    margin-top:170px;
+    margin-bottom:170px;
+    text-align:center;
+    background:rgba(255, 255, 255, 0.83)
```

Comment:
Colon after property should be followed by one space<br>Declaration should be terminated by a semicolon